### PR TITLE
Fixes #7220 ( Extracts some specific effects properties into an element, adds afterimage effect )

### PR DESCRIPTION
**Pull request by Strobia**
# About the pull request

Fixes being able to attack blood spatter with merge  #7220

# Explain why it's good for the game

TERRIBLE BUG BREAKS IMPORTANT GAME MECHANICS

# Testing Photographs and Procedure

</details>

# Changelog

🆑 lolvax1
fix: fixes blood spatter being clickable
/🆑

### CODE REVIEW

**Review by Drulikar — CHANGES_REQUESTED** [review]
Looks good for blood splatter.

However, you will find the same issue with:
`/obj/effect/afterimage`
and
`/obj/effect/shockwave`

Please replicate the same change for them as well

(you can just make another commit to this branch to update the PR)

**Comment by Strobia** [comment]
Sorry if those 2 are wrong I am new :)

File: code/game/objects/effects/shockwave.dm [hunk 1]
```diff
@@ -2,6 +2,8 @@
 //Shockwaves
 //------------------------------------------
 
+mouse_opacity = MOUSE_OPACITY_TRANSPARENT
+
 /obj/effect/shockwave
 	icon = 'icons/effects/light_overlays/shockwave.dmi'
 	icon_state = "shockwave"
```
Comment (Drulikar) [inline code/game/objects/effects/shockwave.dm:9]:
```suggestion
/obj/effect/shockwave
	icon = 'icons/effects/light_overlays/shockwave.dmi'
	icon_state = "shockwave"
	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
```

File: code/game/objects/effects/afterimage.dm [hunk 2]
```diff
@@ -8,6 +8,8 @@
  * - fading_duration: How long it takes for afterimage to fade
  * - fading_to_shift_ratio: How quickly afterimage will reach its target atom before fading (relative to the fading time)
  */
+	mouse_opacity = MOUSE_OPACITY_TRANSPARENT
```
Comment (Drulikar) [inline code/game/objects/effects/afterimage.dm:11]: Please move this to the line after `as_image = TRUE`

**Review by Drulikar — CHANGES_REQUESTED** [review]

**Review by Strobia — COMMENTED** [review]
fixes things hopefully

**Review by Drulikar — APPROVED** [review]

**Pull request merged**